tests/conftest.py

```diff
@@ -1447,6 +1471,45 @@ def titanic_data_context_stats_enabled_config_version_3(tmp_path_factory, monkey
     return ge.data_context.DataContext(context_path)
 
 
+@pytest.fixture(scope="module")
```

Comment:
@lccasagrande For aesthetics, could we please move all fixtures toward the top of the `conftest.py` module (and it would be also easier to read new tests that way)?  Thanks!

Reply:
Done